mail_editor/settings.py

```diff
@@ -57,6 +57,15 @@ def DYNAMIC_CONTEXT(self):
         if dynamic:
             return import_string(dynamic)
 
+    @property
+    def BASE_HOST(self):
+        """
+        protocol, domain and (optional) port, no ending slash
+
+        used to retrieve images for embedding and fix URLs
+        """
+        return getattr(django_settings, 'MAIL_EDITOR_BASE_HOST', "")
```

Comment:
Not sure if we want a separate setting for this or pull it from somewhere else

Reply:
easy enough to do `MAIL_EDITOR_BASE_HOST = BASE_URL` in a project's settings, namespacing is good so I'd stick with this :)